ntfysh_client/SendMessageForm.Designer.cs

```diff
@@ -0,0 +1,108 @@
+﻿namespace ntfysh_client
+{
+    partial class SendMessageForm
```

Comment:
Please add a title to the window so that it doesn't just say "SendMessageForm" in the title bar